packages/bsky/tests/admin/repo-search.test.ts

```diff
@@ -74,7 +74,7 @@ describe('pds admin repo search view', () => {
     expect(res.data.repos[0].did).toEqual(term)
   })
 
-  it('finds repo by email', async () => {
+  it.skip('finds repo by email', async () => {
```

Comment:
Intending to keep this skipped?  If so, might just be worth a little note.

Reply:
yeah it's not clear to me how this will function with multi-pds

it's also not that clear that me that search by email is useful. Altho we may want to add some way to resolve email -> did on the entryway

Reply:
i'll add a @NOTE tho 👍

Reply:
was gonna leave a ntoe, but I just decided to delete it instead. These are appview tests & nothing in the API contract suggests you need to be able to search on email

If this is an important feature, we can find another way of implementing